### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by lschatzkin**

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -20,18 +20,19 @@ <h2 class="black">Defend Our Online Communities</h2>
...
 
-<p>If Congress undermines these important protections by passing SESTA/FOSTA, many online platforms will be forced to place strong restrictions on their users&#8217; speech, silencing a lot of marginalized voices in the process.</p>
+<p>Congress significantly undercut Section 230 and the robust online speech it promotes by passing SESTA/FOSTA. As a result, online platforms must face an awful decision: risk severe civil and criminal penalties for the activities of their users or restrict their users’ speech, silencing a lot of marginalized voices in the process.</p>
 
-<p>Let&#8217;s tell Congress: SESTA/FOSTA won&#8217;t stop sex trafficking, but it <em>will</em> ruin online communities.</p>
+<p>That’s why the Electronic Frontier Foundation has sued the United States Department of Justice [link to “legal challenge” section] to have SESTA/FOSTA declared unconstitutional.</p>
```
Comment (esoterik) [inline layouts/index.html]: there's a leftover TODO here

Comment (lschatzkin) [reply layouts/index.html]: Good spotting! I've fixed it.

File: layouts/index.html [hunk 2]
```diff
@@ -20,18 +20,19 @@ <h2 class="black">Defend Our Online Communities</h2>
...
-<p>If Congress undermines these important protections by passing SESTA/FOSTA, many online platforms will be forced to place strong restrictions on their users&#8217; speech, silencing a lot of marginalized voices in the process.</p>
+<p>Congress significantly undercut Section 230 and the robust online speech it promotes by passing SESTA/FOSTA. As a result, online platforms must face an awful decision: risk severe civil and criminal penalties for the activities of their users or restrict their users’ speech, silencing a lot of marginalized voices in the process.</p>
 
-<p>Let&#8217;s tell Congress: SESTA/FOSTA won&#8217;t stop sex trafficking, but it <em>will</em> ruin online communities.</p>
+<p>That’s why the Electronic Frontier Foundation has <a href="#challenge">sued the United States Department of Justice</a> to have SESTA/FOSTA declared unconstitutional.</p>
 
-            <div class="action-button">
-              <a href="#take-action" class="button">Act Now</a>
+            <div class="action-button button">
+              <a href="#take-action" ><span class="larger">EFF's Legal Case</span><br/>Against SESTA/FOSTA
```
Comment (esoterik) [inline layouts/index.html]: should this link to `#challenge`?

File: layouts/partials/challenge.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  <div class="container text-left">
+    <div class="col-md-8 col-md-offset-2">
+
+      <h2>Legal Challenge to SESTA/FOSTA:<br/>The Woodhill Case</h2>
+      <p>In Woodhull Freedom Foundation et al. v. United States, a wide variety of plaintiffs are asking the court to declare SESTA/FOSTA unconstitutional: two human rights organizations, a digital library, a sex work activist, and a certified massage therapist.</p>
+      <p>SESTA/FOSTA violates the First Amendment by punishing certain types of lawful speech – including speech advocating for the decriminalization of sex work. Its broad wording creates a great deal of uncertainty, both for Internet users and for online platforms about what speech or activity creates liability under the law.</p>
+      <p>The law also violates the Constitution’s prohibition against ex post facto laws—laws that retroactively criminalize activity that was legal at the time that it happened. </p>
+      <p><em>[insert links to newest blog posts on Woodhull]</em></p>
+      <div class="action-button button">
+        <a href="https://eff.org" ><span class="larger">Read More</span><br/>on eff.org
```
Comment (esoterik) [inline layouts/partials/challenge.html]: should this link to `https://www.eff.org/cases/woodhull-freedom-foundation-et-al-v-united-states`?

File: layouts/index.html (hunk 2 above)
Comment (lschatzkin) [reply layouts/index.html]: Yes indeed.

**Pull request merged**